Automattic/woocommerce-payments
# Fix failing E2E tests for BNPL checkout

Fixes https://github.com/Automattic/woocommerce-payments/issues/8354

#### Changes proposed in this Pull Request
This PR fixes the failing BNPL E2E test suite. The currency switcher widget was not appearing on the page when trying to switch the currency to USD, but upon further testing it was established that switching currency isn't needed in this test suite so it got simply removed.

#### Testing instructions

* run [E2E Tests - All](https://github.com/Automattic/woocommerce-payments/actions/workflows/e2e-test.yml) on this branch and confirm tests are passing
* if your local setup allows, run `npm run test:e2e tests/e2e/specs/wcpay/shopper/shopper-bnpls-checkout.spec.js` to confirm it passes

-------------------


**Post merge**
